### ISSUE TITLE
Move Angora from Brawl to a promo set

### ISSUE THREAD

**Issue by jl8e** [buttonmen-dev/buttonmen#2600]
From the wiki: "Angora is an original character by BRAWL: Catfight artist Bryce Nakagawa. The button was printed in small quantities and handed out at conventions. The cup in her hand is a personal trademark of Bryce. Like Sydney, no BRAWL deck of the character exists."

She wasn't released as a Brawl promo, she just is all-speed. Since she's identical to Sydney, we have two identical buttons in the same set,  which is a bit weird.

This was discussed a bit back in #214, but it wasn't really resolved.

**Comment by TheOrgg** [comment]
I'd be fine with folding her and Kasanova into Sydney.

I'd like to see identical buttons combined, with a choice of artwork, personally.  There's a handful out there that exist.

**Comment by jl8e** [comment]
Yeah, that's a completely different and more complicated to implement feature. I'd suggest opening a different feature request for it. (I have thoughts on how it would best be done.)
